apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -4,6 +4,8 @@ import {nameComparator} from '@cdo/apps/util/sort';
 import BlockSvgFrame from '@cdo/apps/blockly/addons/blockSvgFrame';
 import {procedureDefMutator} from './mutators/procedureDefMutator';
 import {BLOCK_TYPES} from '@cdo/apps/blockly/constants';
+import procedureCallerOnChangeMixin from './mixins/procedureCallerOnChangeMixin';
+import {procedureCallerMutator} from './mutators/procedureCallerMutator';
```

Comment:
nit: Is there a reason one of these is exported by default and one isn't? Is that just how Blockly does it?

Reply:
nope this is me being inconsistent, I'll fix this.